.github/workflows/mysqldump_pr.sh

```diff
@@ -0,0 +1,23 @@
+#! /usr/bin/env bash
+
+export GITHUB_USER='edx-deployment'
+export GITHUB_TOKEN=$GH_ACCESS_TOKEN
+export GITHUB_EMAIL='edx-deployment@edx.org'
+export DB_NAME='edxapp'
+
+cd ..
+
+# install hub
+curl -L -o hub.tgz https://github.com/github/hub/releases/download/v2.14.2/hub-linux-amd64-2.14.2.tgz
+tar -zxvf hub.tgz
+
+cd -
```

Comment:
Hi, if you know the name of the directory here, could you change this to `cd <name of directory>`. That would be cleaner to read

Reply:
Thanks, @jinder1s I have updated PR to use var with repository name to change directory.